pkg/license/license.go

```diff
@@ -10,13 +10,41 @@ import (
 	kotsv1beta1 "github.com/replicatedhq/kots/kotskinds/apis/kots/v1beta1"
 	"github.com/replicatedhq/kots/kotskinds/client/kotsclientset/scheme"
 	"github.com/replicatedhq/kots/pkg/buildversion"
+	"github.com/replicatedhq/kots/pkg/logger"
+	"github.com/replicatedhq/kots/pkg/store"
 )
 
 type LicenseData struct {
 	LicenseBytes []byte
 	License      *kotsv1beta1.License
 }
 
+func ResolveExistingLicense(verifiedLicense *kotsv1beta1.License) (bool, error) {
```

Comment:
let's call it "newLicense" instead just to be clearer in the context of this function?
```suggestion
func ResolveExistingLicense(newLicense *kotsv1beta1.License) (bool, error) {
```